src/overlays/actors/ovl_En_Skb/z_en_skb.c

```diff
@@ -95,129 +162,977 @@ const ActorInit En_Skb_InitVars = {
+        if (this->actor.speedXZ < 0.0f) {
+            this->actor.speedXZ += 0.05f;
+        }
+    }
+
+    if ((this->actor.colorFilterTimer == 0) && (this->actor.bgCheckFlags & 1)) {
+        if (this->unk_3DC == 0) {
+            if (this->actor.colChkInfo.health == 0) {
+                func_809961E4(this, globalCtx);
+            } else {
+                func_80994DA8(this, globalCtx);
+            }
+        } else {
+            switch (this->unk_3D6) {
+                case 2:
+                    func_809952D8(this);
+                    break;
+
+                case 3:
+                    func_809953E8(this);
+                    break;
+            }
+        }
+    } else if ((this->actor.colorFilterTimer == 0) && (this->unk_3DC == 1) && (this->unk_3D6 == 4)) {
+        this->unk_3DE = 11;
+        this->actionFunc = func_8099556C;
+    }
+}
+
+void func_80995F98(EnSkb* this) {
+    if ((this->unk_3DE == 9) || (this->unk_3DE == 0xA)) {
+        this->actor.world.rot.y = this->actor.yawTowardsPlayer;
+        func_800BDC5C(&this->skelAnime, sAnimations, 8);
+        this->actor.gravity = -1.0f;
+        this->actor.speedXZ = 1.0f;
+    } else if (this->unk_3DE == 0xB) {
+        func_800BDC5C(&this->skelAnime, sAnimations, 9);
+        this->actor.speedXZ = 3.2f;
+        this->actor.velocity.y = 2.0f;
+        this->actor.gravity = -1.0f;
+    } else {
+        this->actor.world.rot.y = this->actor.yawTowardsPlayer;
+        func_800BDC5C(&this->skelAnime, sAnimations, 3);
+        if (this->actor.bgCheckFlags & 1) {
+            this->actor.speedXZ = -4.0f;
+        }
+    }
+    Audio_PlayActorSound2(&this->actor, NA_SE_EN_STALKID_DAMAGE);
+    this->unk_3DE = 6;
+    this->actionFunc = func_809960AC;
+}
+
+void func_809960AC(EnSkb* this, GlobalContext* globalCtx) {
+    if (this->actor.bgCheckFlags & 2) {
+        s32 i;
+
+        for (i = 0; i < 10; i++) {
+            func_809947B0(globalCtx, this, &this->actor.world.pos);
+        }
+        this->actor.speedXZ = 0.0f;
+    }
+
+    if (this->actor.bgCheckFlags & 1) {
+        if (this->actor.speedXZ < 0.0f) {
+            this->actor.speedXZ += 0.05f;
+        }
+        Math_SmoothStepToS(&this->actor.shape.rot.y, this->actor.yawTowardsPlayer, 1, 0x9C4, 0);
+    }
+
+    Math_SmoothStepToS(&this->actor.shape.rot.x, 0, 0x10, 0x7D0, 0x64);
+    if (func_801378B8(&this->skelAnime, this->skelAnime.animFrameCount) && (this->actor.bgCheckFlags & 1)) {
+        this->actor.shape.rot.x = 0;
+        this->actor.world.rot = this->actor.shape.rot;
+        func_80994DA8(this, globalCtx);
+    }
+}
+
+void func_809961E4(EnSkb* this, GlobalContext* globalCtx) {
+    func_800BDC5C(&this->skelAnime, sAnimations, 4);
+    this->unk_3D8 |= 0x40;
+    if (this->actor.bgCheckFlags & 1) {
+        this->actor.speedXZ = -6.0f;
+    }
+    this->unk_3E4 = 0;
+    this->actor.flags &= ~1;
+    func_800F0568(globalCtx, &this->actor.world.pos, 0x28, 0x3833);
```

Comment:
```suggestion
    func_800F0568(globalCtx, &this->actor.world.pos, 40, NA_SE_EN_STALKID_DEAD);
```
You should also be aware that this function is changing names to `Audio_PlaySoundAtPosition` in https://github.com/zeldaret/mm/pull/250/files, this might have to change names if that goes in first